TTHAnalysis/python/plotter/susy-ewkino/3l/plots_ewkino.txt

```diff
@@ -1,115 +1,111 @@
 
-## baseline region
+## baseline region per final state
 ## ----------------------------------------------------------
-BR          : BR            : 3,-0.5,2.5  ; XTitle="Baseline Region", Legend="TR", LegendCutoff=1e-5, xBinLabels="none\,3l\,>3l"
+BR          : BR            : 9,0.5,9.5   ; XTitle="Baseline Region", Wide=True, Legend="TR", LegendCutoff=1e-5, xBinLabels="A\,B\,C\,D\,E\,F\,G\,H\,I"
 
-## signal regions
+## signal regions per final state
 ## ----------------------------------------------------------
-SR_3l_light : SR            : 36,0.5,36.5 ; XTitle="Signal Region", Legend="TR", IncludeOverflows=False, LegendCutoff=1e-5
-SR_3l_1tau  : SR-36         : 84,0.5,84.5 ; XTitle="Signal Region", Legend="TR", IncludeOverflows=False, LegendCutoff=1e-5 
-SR_3l_2tau  : SR-120        : 60,0.5,60.5 ; XTitle="Signal Region", Legend="TR", IncludeOverflows=False, LegendCutoff=1e-5 
-SR_4l       : SR-144        : 12,0.5,12.5 ; XTitle="Signal Region", Legend="TR", IncludeOverflows=False, LegendCutoff=1e-5
+SR_3l_light : SR            : 42,0.5,42.5 ; XTitle="Signal Region"  , Wide=True, Legend="TR", IncludeOverflows=False, LegendCutoff=1e-5
```

Comment:
IncludeOverflows should not be used in the plot file, this is dangerous when then running the datacard where the plot file is not read. Please remove all IncludeOverflows from the plot file, and replace them with appropriate cuts added to the plot command like:
-A acut removeoverflow 'SR>=1 && SR<=43'